content-resources/src/main/java/flyway/oskari/V1_43_4__register_download_basket_bundle.java

```diff
@@ -0,0 +1,24 @@
+package flyway.downloadbasket;
+
+import java.sql.Connection;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import fi.nls.oskari.db.BundleHelper;
+import fi.nls.oskari.domain.map.view.Bundle;
+
+public class V1_43_4__register_download_basket_bundle implements JdbcMigration{
```

Comment:
This is the first ever flyway-migration for the module "downloadbasket" and it starts with version 1.43.4? Should this be version 1.0 or located in the "oskari" module?

Reply:
Oh, sorry. Actually the other "V1_0_12__add_download_basket_to_views" in downloadbasket module is run before this as both are in the same module "downloadbasket". So the migration that tries to add the bundle to a view is run before the bundle is being registered with this as 1.0.12 < 1.43.4. This migration won't be run as the 1.0.12 will fail since it can't find the bundle id. If you want to do this as a new flyway-module please make sure the version numbers are not all over the place.